inst/stan/functions/construct_aux_rt.stan

```diff
@@ -15,19 +15,13 @@ vector construct_aux_rt(vector log_state_rt,
 
   // presets
   int n_time = dims(state_deviation_noise_vec)[1];
-  vector[n_time] log_aux_site_rt;
-  real state_deviation_t_i = 0;
-  // To explain state_deviation_t_i = 0 initialization,
-  // we need a state_deviation_t_i value for t_1 = 1, so we will
-  // initialize the process with index 1 of spatial
-  // deviation noise matrix w/out spatial_deviation ar coeff, so we set
-  // state_deviation_t_i to 0 initially.
-
-  for (t_i in 1:n_time) {
-    state_deviation_t_i = state_deviation_ar_coeff * state_deviation_t_i +
-        state_deviation_noise_vec[t_i];
-    log_aux_site_rt[t_i] = log_state_rt[t_i] + state_deviation_t_i;
-  }
+  vector[n_time] log_aux_site_rt = ar1(
+    log_state_rt,
+    state_deviation_ar_coeff,
+    1,
+    state_deviation_noise_vec,
+    0
```

Comment:
1. I'd make the `state_deviation_noise_vec` into the standard normal rvs and pass the state deviation s.d. separately to `ar1()`
2. I'd make the init at stationarity option configurable in the `construct_aux_rt` function, and pass it through to `ar1()`